stubs/pyxdg/xdg/IconTheme.pyi

```diff
@@ -0,0 +1,59 @@
+from _typeshed import Unused
+from collections.abc import Collection
+from os import PathLike
+from typing import AnyStr
+
+from xdg.IniFile import IniFile
+
+basedir: Unused  # This is something that shouldn't be included since it's
+
+class IconTheme(IniFile):
+    def __init__(self) -> None: ...
+    dir: str
+    name: str
+    comment: str
+    directories: list[str]
+    type: str
+    def parse(self, file: PathLike[AnyStr]) -> None: ...  # type: ignore[override]
```

Comment:
I don't think this works with bytes paths. `self.name` comes out of the file through some `os.path` operations, and it's returned from `__repr__`, but `__repr__` is only allowed to return `str`.
```suggestion
    def parse(self, file: PathLike[str]) -> None: ...  # type: ignore[override]
```